torch/csrc/jit/codegen/cuda/inline_propagator.cpp

```diff
@@ -11,21 +11,24 @@ namespace fuser {
 namespace cuda {
 
 bool InlinePropagatorSelector::allowC2P(TensorView* from, TensorView* to) {
-  return selected_.count(to) > 0;
+  return propagate_all_ || selected_.count(to) > 0;
```

Comment:
Is this `propagate_all_` used? If you want to select all, you could just pass nullptr to the `selector` argument of the ctor of `MaxRootDomainInfoSpanningTree`:
https://github.com/csarofeen/pytorch/blob/942be5b256056d0e02877361b814ae6af32ca15f/torch/csrc/jit/codegen/cuda/maxinfo_propagator.h#L231-L233